start-a-local-cluster.md

```diff
@@ -32,34 +34,38 @@ Data is stored on-disk and client/server communication is secure. Setup involves
     ```
 
 3. [Run some queries](basic-sql-commands.html).
-4. Check out the Admin UI by pointing your browser to `http://<your local host>:26257`. You can find your local host by running `hostname` in your shell.    
-
-## Standard Mode
 
-1. From the directory containing the `cockroach` binary, initialize the cluster:
+4. Check out the Admin UI by pointing your browser to `http://<your local host>:26257`. You can find your local host by running `hostname` in your shell.    
 
+5. To simulate a multi-node cluster, add each new node as follows:
+    
     ```bash
-    $ ./cockroach init --stores=ssd=dev/data
+    $ ./cockroach start --dev --port=26258 --pgport=15433 --join=<your local host>:26257
```

Comment:
Thanks for catching that, @spencerkimball. Changed **Dev Mode** back to single-node only, changed **Secure Mode** back to **Standard Mode**, and tweaked intro a bit.

Let me know if you see anything else, or if we're good to merge.